documentation/overview.md

```diff
@@ -1002,20 +1002,30 @@ Please note that the denominator we use is the living wage for a single full-tim
 
 ### Overview
 
-* **Analyst & Programmer:** Tina Chelidze
-* **Year(s):** 2021
-* **Final data name(s):** `digital_access_county_2021.csv` and `digital_access_city_2021.csv'
-* **Data Source(s):** Census Bureau's American Community Survey 
-* **Notes:**
-* **Data Quality Index:** `1` means the observation count informing the estimate is greater than or equal to 30. The quality index is a `2` otherwise.
+* **Analyst & Programmer:** Tina Chelidze and Manuel Alcalá Kovalski
+* **Year(s):** 2016, 2017, 2018, 2019, 2021, 2022
```

Comment:
it looks like your output files have 2020 as well?

Reply:
Thanks for catching this! It looks like it was included in the output but all values were NA. So I now filter 2020 before exporting!